docs/log_collection.md

```diff
@@ -24,7 +24,7 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
     }
     ```
 
-2. Initialize the library with your application context and your [Datadog client token][2]. For security reasons, you must use a client token: you cannot use [Datadog API keys][3] to configure the `dd-sdk-android` library as they would be exposed client-side in the Android application APK byte code. For more information about setting up a client token, see the [client token documentation][2]:
+2. Initialize the library with your application context and your [Datadog client token][2]. You also need to provide an Application ID (see our [RUM Getting Started page][6]). For security reasons, you must use a client token: you cannot use [Datadog API keys][3] to configure the `dd-sdk-android` library as they would be exposed client-side in the Android application APK byte code. For more information about setting up a client token, see the [client token documentation][2]:
```

Comment:
I'm not sure this clarifies which Application ID you're talking about. The link is going to a page that doesn't talk about application IDs at all. What does "application context" in the first sentence mean, if not application ID? Could we rewrite this as "Initialize the library with the application ID and Datadog client token"? I'm still not sure that tells them what application ID to use.

Reply:
@kayayarai Maybe it's better to then use the same language from https://docs.datadoghq.com/real_user_monitoring/android/?tab=us; I got confirmation that it is the same process to go through and generate the application_id and client_token. Overall I want the customers to know they can generate both of these fields by going to UX Monitoring > RUM Applications > New Application. Once there they need to choose and application name, click the 'generate client token', and switch the tab over to the android tab. (Do you think it's better to include of this in the doc?

Reply:
Yeah, I don't think it hurts to spell it out like that.